gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/engine/GfxdConstants.java

```diff
@@ -797,6 +797,13 @@
   final String GFXD_ENABLE_GETALL_LOCALINDEX_EMBED_GFE = GFXD_PREFIX
       + Attribute.ENABLE_GETALL_LOCALINDEX_EMBED_GFE;
 
+  /**
+   * System property to certain store queries to spark for better performance
+   */
+
+  final String GFXD_ROUTE_SELECTED_STORE_QUERIES_TO_SPARK =
```

Comment:
Can we have a simpler/shorter name like:

"enable-routing-arbiter" or "enable-routing-manager"

as this is short and also other properties have things like ( enable/disable )